pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -199,6 +205,7 @@ buildStdenv.mkDerivation ({
     which
     wrapGAppsHook
   ]
+  ++ lib.optionals pgoSupport [ xvfb-run ]
```

Comment:
```suggestion
  ++ lib.optional pgoSupport xvfb-run
```

Reply:
I really prefer optionals as it makes it easier to add further items to the list when necessary.